tests/bluetooth/tester/src/btp/btp_ias.h

```diff
@@ -0,0 +1,15 @@
+/* btp_ias.h - Bluetooth tester headers */
+
+/*
+ * Copyright (c) 2022 Codecoup
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "zephyr/bluetooth/services/ias.h"
+
+/* events */
+#define IAS_EV_OUT_ALERT_ACTION       0x80
+struct ias_alert_action_ev {
+	enum bt_ias_alert_lvl alert_lvl;
```

Comment:
this should be uint8_t
and btp should define meaning for values 0x00 0x01 0x02 

than in zephyr we can map enum (or cast if those match) to those values

Reply:
done